src/twisted/web/server.py

```diff
@@ -34,7 +34,14 @@
 from twisted.spread.pb import Copyable, ViewPoint
 from twisted.web import http, iweb, resource, util
```

Comment:
Do we still want to keep the `http` import  here ? Just asking :)
 
I guess that this is here to simplify the access to the HTTP response code.

We can leave this PR as it is.

In the future, maybe we can have a separare "collection" object for the response codes, other than the `twisted.web.http` module itself.

```suggestion
from twisted.web import iweb, resource, util
```

Reply:
It's used in a few places still, I didn't remove it from places where it was less common.